pyproject.toml

```diff
@@ -22,6 +22,33 @@ extend_skip_glob = ["**/migrations/*"]
 
 ##############################################
 
+[tool.ruff]
+target-version = "py310"
+line-length = 120
+select = ["F", "E", "B", "W", "N", "UP", "YTT", "FIX", "ASYNC", "A", "DJ", "EXE", "ICN", "G", "SLOT", "TID", "TCH", "INT", "C4", "ISC", "INP", "PIE", "RSE", "RET", "COM","PGH","FLY", "PERF", "PLE"]  # PYI,Q,SIM,PTH,PL,TRY,RUF
+ignore = [
+    "E501",                  # line-too-long: black does code formatting for us
+    "FIX004",                # hacks should be possible
+    "A003",                  # we shadow some builtins sometime
+    "DJ008",                 # do not use __str__ where it is not required
+    "COM812",                # incompatible with formatter
+    "PERF203",
```

Comment:
I wouldn't expect us to hit this one often -- can we enable it?

Reply:
there were some occurrences, but I can try

Reply:
If there's multiple occurrences or something annoying about them, I'm also fine to keep the exclusion and maybe add a comment why we weren't able to enable it. Just off the top of my head I just didn't see why we would often violate this.

Reply:
Okay, _some_ occurrences mean one, in 
https://github.com/e-valuation/EvaP/blob/75bee4190544aeab13861560536f8939f6965356/evap/evaluation/models.py#L929-L956
only used as a management command. Here, the try-except covers invalid state transitions. We can add a `noqa` here